packages/core/cli/start.js

```diff
@@ -1,29 +1,32 @@
 /* eslint-disable global-require, no-console, import/order */
-const express = require('express');
-const proxy = require('http-proxy-middleware');
-const cookiesMiddleware = require('universal-cookie-express');
+// Get environmental variables
+const getEnv = require('../config/env');
+const {
+  API_ROOT_URL,
+  API_ORIGN,
+  NODE_ENV,
+} = getEnv();
```

Comment:
Will these be ok without defaults?

Reply:
API_ORIGIN will fall back to API_ROOT_URL, and yeah I figure the app _should_ break if that env var isn't set, not sure what it would even look like if it wasn't.